### PR TITLE
Change to documentation

### PR DESCRIPTION
**Pull request by Bhargavishnu**
### **Describe your change:**

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,28 @@
 ![](https://img.shields.io/github/repo-size/TheAlgorithms/Python.svg?label=Repo%20size&style=flat-square)&nbsp;
 <!--[![Tested on Python 3.7](https://img.shields.io/badge/Tested%20-Python%203.7-blue.svg?logo=python&style=flat-square)]( https://www.python.org/downloads) &nbsp;-->
 
+
+### Description
+
+This repository contains all the algorithms implemented in Python
```
Comment (cclauss) [inline README.md]: `all the` is not required.  If all of the algorithms were here then we would not accept new ones.  ;-)

File: README.md [hunk 2]
```diff
@@ -8,6 +8,28 @@
 ![](https://img.shields.io/github/repo-size/TheAlgorithms/Python.svg?label=Repo%20size&style=flat-square)&nbsp;
 <!--[![Tested on Python 3.7](https://img.shields.io/badge/Tested%20-Python%203.7-blue.svg?logo=python&style=flat-square)]( https://www.python.org/downloads) &nbsp;-->
 
+
+### Description
+
+This repository contains all the algorithms implemented in Python 
+Just copy search the repository for the algorithm you want to implement and use
```
Comment (cclauss) [inline README.md]: `copy` is not required in its current location.

`Just search the repository for the algorithm(s) you want to use and copy them.  You can also improve these algorithms or add new ones by creating pull requests.`

File: README.md [hunk 3]
```diff
@@ -8,6 +8,28 @@
...
+
+### Requirements
+
+Requirements for the codes are [here](requirements.txt).
+Install requirements by:<br>
+
+Python 2
+<pre>pip install -r requirements.txt</pre>  
+or 
+<pre>python -m install -r requirements.txt </pre>
```
Comment (cclauss) [inline README.md]: Please delete lines 22 to 25.  Legacy Python is [dead](https://pythonclock.org) and this repo stopped supporting in a long time ago.  (See CONTRIBUTING.md)

File: README.md [hunk 4]
```diff
@@ -8,6 +8,28 @@
...
+
+Python 3
+<pre>pip3 install -r requirements.txt </pre>
```
Comment (cclauss) [inline README.md]: `<pre>python3 -m pip install -r requirements.txt </pre>`

Backticks \`\` can be used instead of `<pre>` in markdown.

Comment (Bhargavishnu) [reply README.md]: Thank you i learned some new stuffs

File: README.md (hunk 3 above)
Comment (Bhargavishnu) [reply README.md]: sure

File: README.md (hunk 2 above)
Comment (Bhargavishnu) [reply README.md]: yes

**Comment by onlinejudge95** [comment]
Needs the command for starting a `virtual environment` before the installation of dependencies is done.

`python3 -m venv env` or `virtualenv env`

**Comment by itsvinayak** [comment]
I think, we should close this PR

**Pull request closed without merging**